cypress/features/regression/accessibility/accessibilityCommon.feature

```diff
@@ -87,3 +86,4 @@ Scenario Outline: Component <component> basic story
       | I18nComponent |
       | Link          |
       | MenuList      |
+    # | Pages         |
```

Comment:
Do we need this ?

Reply:
Igor asked me to move Pages to this section. It was part of an `@ignore` block but the path to find the component's test story was incorrect. I've moved it to this location along with the rest of the components that don't have classic theme just in case Portal is refactored and can be tested in Cypress.